### PR TITLE
Fix the browser node and ensure it doesn't colour the informant output

### PR DESCRIPTION
**Pull request by expenses**
Some `std::time::Instant`s got added which broke things again. Replacing the `imformant_prefix` in the service builder with a `sc_informant::OutputFormat` in the config seems like a much better idea because the informant prefix isn't a generic.

### CODE REVIEW

**Comment by bkchr** [comment]
What happened to the browser test?

File: client/cli/src/config.rs [hunk 1]
```diff
@@ -477,6 +477,7 @@ pub trait CliConfiguration: Sized {
 			announce_block: self.announce_block()?,
 			role,
 			base_path: Some(base_path),
+			informant_output_format: sc_informant::OutputFormat::default(),
```
Comment (cecton) [inline client/cli/src/config.rs:480]: That must go to a method that will provide the default by default like every other thing in this trait

**Comment by expenses** [comment]
@bkchr because it's not connecting to a node, it doesn't run a whole lot of logic, and doesn't pick up these panics.

File: primitives/io/src/batch_verifier.rs [hunk 2]
```diff
@@ -160,7 +160,7 @@ impl BatchVerifier {
 	#[must_use]
 	pub fn verify_and_clear(&mut self) -> bool {
 		let pending = std::mem::take(&mut self.pending_tasks);
-		let started = std::time::Instant::now();
+		let started = wasm_timer::Instant::now();
```
Comment (NikVolf) [inline primitives/io/src/batch_verifier.rs:163]: Panic here means that light node verifies full block, doesn't it?

Is it a correct behaviour?

File: client/cli/src/config.rs [hunk 3]
```diff
@@ -405,6 +405,13 @@ pub trait CliConfiguration: Sized {
 		Ok(true)
 	}
 
+	/// Get the format for the informant
+	///
+	/// By default this enables colour and does not use a prefix.
+	fn informant_output_format(&self) -> sc_informant::OutputFormat {
```
Comment (cecton) [inline client/cli/src/config.rs:411]: It needs to be a Result to allow the developer who will override to put failing logic if they want.

File: client/cli/src/config.rs [hunk 4]
```diff
@@ -405,6 +405,13 @@ pub trait CliConfiguration: Sized {
 		Ok(true)
 	}
 
+	/// Get the format for the informant
+	///
+	/// By default this enables colour and does not use a prefix.
+	fn informant_output_format(&self) -> Result<sc_informant::OutputFormat> {
```
Comment (bkchr) [inline client/cli/src/config.rs:411]: @cecton this does not makes any sense and `sc_informant` should also not be a dependency of `sc-cli`.

It was also not used anymore by `sc-cli`.

Why should an option that is not controlled by the CLI, be part of the `CliConfiguration`?

**Comment by bkchr** [comment]
> @bkchr because it's not connecting to a node, it doesn't run a whole lot of logic, and doesn't pick up these panics.

Can we please just run a `--dev` node and make the light client connect to this node?

File: primitives/io/src/batch_verifier.rs (hunk 2 above)
Comment (expenses) [reply primitives/io/src/batch_verifier.rs]: No, that was not mean to be there.

**Comment by expenses** [comment]
@bkchr I'll look into this.

File: client/cli/src/config.rs (hunk 4 above)
Comment (cecton) [reply client/cli/src/config.rs:411]: CliConfiguration has a bad name but it is the trait that transforms anything into a sc-service::Configuration

Comment (cecton) [reply client/cli/src/config.rs:411]: Ok no that changed during the development and we added the shared_params and other params so I guess CliConfiguration is specialized for CLI.

Comment (cecton) [reply client/cli/src/config.rs:411]: So you can drop it I guess @expenses

Comment (expenses) [reply client/cli/src/config.rs:411]: Ok :+1:

**Pull request merged**